### PR TITLE
Documentation: complete Enumerator tables

### PR DESCRIPTION
**Pull request by zjiaqi2018**
Trying to complete the Enumerator tables here: 
https://www.dealii.org/developer/doxygen/deal.II/namespaceTimeStepping.html#abff97b5326e452552f108a379dd6cff4
Not sure if there is a better way to do it. 

This is what it looks like now: 
Screenshot from 2020-09-23 17-43-51

### CODE REVIEW

File: include/deal.II/base/time_stepping.h [hunk 1]
```diff
@@ -78,8 +119,17 @@ namespace TimeStepping
    */
   enum embedded_runge_kutta_time_step
   {
+    /**
```
Comment (tjhei) [inline include/deal.II/base/time_stepping.h:122]: Can you delete the descriptions of the 3 entries in the comment block above? No reason to exactly duplicate the information.

Comment (zjiaqi2018) [reply include/deal.II/base/time_stepping.h]: I thought every Enumerator needs a table, sorry.

Comment (tjhei) [reply include/deal.II/base/time_stepping.h]: Sorry, you misunderstood: please put the descriptions of the enum values back in, but remove the identical and now redundant text in the documentation 3 lines above the line I commented at. (Github doesn't want me to comment on a line you did not change)

Comment (zjiaqi2018) [reply include/deal.II/base/time_stepping.h]: That means we will move the descriptions in parenthesis to the table, correct?

Comment (tjhei) [reply include/deal.II/base/time_stepping.h]: Yes, exactly like you had it before.

Comment (tjhei) [reply include/deal.II/base/time_stepping.h:120]: this is what I meant, thanks.

**Review by drwells — APPROVED** [review]

**Review by kronbichler — APPROVED** [review]

**Pull request merged**